internal/resource/testing/resource-testing.go

```diff
@@ -51,6 +51,14 @@ func NewDeviceMock(migEnabled bool) *DeviceMock {
 		IsMigEnabledFunc:     func() (bool, error) { return migEnabled, nil },
 		IsMigCapableFunc:     func() (bool, error) { return migEnabled, nil },
 		GetMigDevicesFunc:    func() ([]resource.Device, error) { return nil, nil },
+		GetPCIClassFunc:      func() (uint32, error) { return 0x030000, nil },
```

Comment:
nit: Should this return `0` by default since it's "unknown" / "undefined"?

Reply:
Fixed